src/Api/Controllers/AccountsController.cs

```diff
@@ -213,6 +214,13 @@ public async Task PostSetPasswordAsync([FromBody]SetPasswordRequestModel model)
             var result = await _userService.SetPasswordAsync(model.ToUser(user), model.MasterPasswordHash, model.Key);
             if (result.Succeeded)
             {
+                if (!model.OrgIdentifier.IsNullOrEmpty())
+                {
+                    var acceptResult =
+                        await _organizationService.AcceptUserAsync(model.OrgIdentifier, model.ToUser(user),
+                            _userService);
+                }
```

Comment:
This is business logic that should exist in the `SetPasswordAsync` service.

Reply:
Moved into the `UserService.SetPasswordAsync` - made adjustments to interface/method params as necessary